cpp/src/parquet/bloom_filter.h

```diff
@@ -167,6 +167,31 @@ class PARQUET_EXPORT BloomFilter {
 
   virtual ~BloomFilter() = default;
 
+  // Variant of const reference argument to facilitate template
+
+  /// Compute hash for ByteArray value by using its plain encoding result.
+  ///
+  /// @param value the value to hash.
+  uint64_t Hash(const ByteArray& value) const { return Hash(&value); }
```

Comment:
could we move these to be next to there equivelant non-const function calls?

Reply:
It's ok, but currently I put const reference argument together...I'll change it if you like

Reply:
It seems they are better to be relocated to below line 107.